### PR TITLE
Alex's suggestions

### PR DESCRIPTION
**Pull request by AlexLipp**

### CODE REVIEW

File: packages/get_record.py [hunk 1]
```diff
@@ -1,190 +1,152 @@
 import pandas as pd
-import numpy as np
```
Comment (jonasyip) [inline packages/get_record.py:2]: ok

File: packages/get_record.py [hunk 2]
```diff
@@ -1,190 +1,152 @@
...
-from datetime import datetime
```
Comment (jonasyip) [inline packages/get_record.py:3]: change

File: packages/get_record.py [hunk 3]
```diff
@@ -1,190 +1,152 @@
...
 
-class record_search:
```
Comment (jonasyip) [inline packages/get_record.py:5]: test

File: packages/get_record.py [hunk 4]
```diff
@@ -1,190 +1,152 @@
...
-    def __init__(self):
-        self.height_record_path = ""
-        self.flow_record_path = ""
```
Comment (jonasyip) [inline packages/get_record.py:8]: add sth

**Review by jonasyip — CHANGES_REQUESTED** [review]

File: packages/get_record.py [hunk 5]
```diff
@@ -2,36 +2,64 @@
...
+        Get path to .csv height file.
+
+        Returns
+        -------
+        height_record_path : string
+            File or url path.
 
-    def set_height_record_path(self, record_path):
+        """
+        return self._height_record_path
```
Comment (jonasyip) [inline packages/get_record.py:25]: setter

File: packages/get_record.py [hunk 6]
```diff
@@ -55,36 +83,36 @@ def find_height_record(self, date, time, return_all=True):
...
                 return exact_search
 
-        #Hour seach
+        # Hour seach
         else:
             search_datetime_object = datetime.fromisoformat(search_dateTime)
 
             search_dateTime_hour = "%sT%s" % (date, time[0:2])
-            hour_search = df_height_record['dateTime'].str.contains(search_dateTime_hour)
+            hour_search = df_height_record["dateTime"].str.contains(search_dateTime_hour)
```
Comment (jonasyip) [inline packages/get_record.py]: using double quotation mark

File: packages/get_record.py [hunk 7]
```diff
@@ -95,22 +123,23 @@ def find_height_record(self, date, time, return_all=True):
...
-                record_dateTime_object = datetime.fromisoformat(record['dateTime'])
+                record_dateTime_object = datetime.fromisoformat(record["dateTime"])
                 diff = search_datetime_object - record_dateTime_object
                 time_diff_list.append(abs(diff.total_seconds()))
 
+            assert len(df_hour_record_indices) == len(
+                time_diff_list
+            )  # Index list should equal to the number of loops
 
-            assert len(df_hour_record_indices) == len(time_diff_list) #Index list should equal to the number of loops
```
Comment (jonasyip) [inline packages/get_record.py:103]: don't assert

**Review by jonasyip — APPROVED** [review]

**Comment by AlexLipp** [comment]
Awesome!

**Pull request merged**